src/include/storage/index/hash_index_builder.h

```diff
@@ -62,13 +63,15 @@ class HashIndexBuilder final : public InMemHashIndex {
     // Reserves space for at least the specified number of elements.
     void bulkReserve(uint32_t numEntries) override;
 
-    bool append(T key, common::offset_t value);
-    bool lookup(T key, common::offset_t& result);
+    using KeyView =
```

Comment:
I think `Key` is fine here. Same from this comment https://github.com/kuzudb/kuzu/pull/2997#discussion_r1521748116